public/index.html

```diff
@@ -86,6 +89,9 @@ <h2>oAuth info</h2>
         var access_token = params.access_token,
             refresh_token = params.refresh_token,
             error = params.error;
+            
+        sessionStorage.setItem("accessToken", params.access_token);
```

Comment:
Duplication of setting access_token here. I like this idea of setting/retrieving tokens from/in session storage. Let's standardize this throughout the app! 

```suggestion
        sessionStorage.setItem("access_token", params.access_token);
        sessionStorage.setItem("refresh_token", params.access_token);
        sessionStorage.setItem("error", params.error);
```